content/shared/influxdb3-query-guides/snapshots/snowflake.md

```diff
@@ -0,0 +1,152 @@
+# Integrating InfluxDB with Snowflake Using Apache Iceberg
+
+## Overview
+
+Snapshots for Snowflake enable users to export time-series data from InfluxDB into a structured format using Apache Iceberg. This integration facilitates efficient data sharing between InfluxDB and Snowflake without the need for complex ETL processes.
```

Comment:
```suggestion
Export time-series data snapshots from InfluxDB into Apache Iceberg format.
Integrate data with Snowflake and other Iceberg-compatible tools without the need for complex ETL processes.
```

- "Snapshots for Snowflake" suggests that this is the feature name.
Let's keep Snowflake prominent, but primarily emphasize exporting in Iceberg format (which most of the page does).
- Use second person
- Avoid big words like "facilitates"